packages/react-core/src/components/Label/Label.tsx

```diff
@@ -91,7 +90,8 @@ export const Label: React.FunctionComponent<LabelProps> = ({
   ...props
 }: LabelProps) => {
   const [isEditableActive, setIsEditableActive] = useState(false);
-  const editableDivRef = React.createRef<HTMLDivElement>();
+  const editableDivRef = React.createRef<HTMLButtonElement>();
+  const editableInputRef = React.createRef<HTMLInputElement>();
```

Comment:
Will it be better to use `React.useRef` here so that refs will not be re-created on every render?